util/python/BUILD

```diff
@@ -1,7 +1,17 @@
 # This is a placeholder for python headers. Projects needing to use
 # fast cpp protos in protobuf's python interface should build with
 # --define=use_fast_cpp_protos=true, and in addition, provide
-# //util/python:python_headers dependency that in turn provides Python.h.
+# @python//:python_headers dependency that in turn provides Python.h.
```

Comment:
Shall we also bind this to //external, so we have a uniformed dependency style. Also, name it with protobuf prefix to avoid conflicts with other projects.